app-sso/getting-started/appsso-overview.hbs.md

```diff
@@ -5,9 +5,9 @@ Single Sign-On (commonly called AppSSO).
 
 Use this topic to learn how to:
 
-1. [Set up your first authorization server](provision-auth-server.hbs.md).
-1. [Provision a ClientRegistration](client-registration.hbs.md).
-1. [Deploy an application](application.hbs.md) that uses the provisioned ClientRegistration to enable SSO.
+1. [Set up your first simplistic authorization server](./provision-auth-server.hbs.md).
+1. [Claim credentials to connect your workload](./claim-credentials.hbs.md)
+1. [Deploy your workload, a.k.a application](./application.hbs.md)
```

Comment:
**[suggestion, non-blocking]:** This might be simpler 
```suggestion
1. [Deploy your workload](./application.hbs.md)
```

Reply:
Ok! I was trying to put myself in the shoes of an application developer and thinking that `Workload` is a pretty vmware specific term. I thought this could help them understand.